app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt

```diff
@@ -335,99 +370,42 @@ class MainActivity : SimpleActivity() {
         val finalSeconds = seconds % 60
         val parts = mutableListOf<String>()
```

Comment:
you can use `buildList`

Reply:
Yeah, sorry about these. There are a lot of parts of the code I didn't really have to change, but I missed opportunities to clean them up.